ruby/lib/ci/queue/redis/build_record.rb

```diff
@@ -56,20 +56,29 @@ def record_warning(type, attributes)
           redis.rpush(key('warnings'), Marshal.dump([type, attributes]))
         end
 
+        Test = Struct.new(:id) # Hack
```

Comment:
We should change `#acknowledge` to take the raw ID string.

Reply:
I will follow up on that.